assets/js/modules/search-console/index.js

```diff
@@ -84,4 +90,17 @@ domReady( () => {
 			AREA_PAGE_DASHBOARD_SEARCH_FUNNEL,
 		],
 	);
+	Widgets.registerWidget(
+		'searchConsolePopularKeywords',
+		{
+			component: DashboardPopularKeywordsWidget,
+			width: Widgets.WIDGET_WIDTHS.HALF,
+			priority: 1,
+			wrapWidget: true,
```

Comment:
Since the table in the widget has its own padding etc, it should not use `wrapWidget: true` and instead pass `wrapWidget: false`. Then, the actual component above needs to take care of rendering the `Widget` component itself - for the regular case, where data is available, it should then add the class so that there's no padding on the widget itself. (For the other cases (error, loading, zero data), it can just render the `Widget` component as is, since for those cases the regular padding is needed.)

See the implementation of the new PageSpeed Insights widget to see an example for a widget without `wrapWidget: true`.

Reply:
updated